src/custom/state/lists/hooks/hooksMod.ts

```diff
@@ -194,3 +201,48 @@ export function useIsListActive(url: string): boolean {
   const activeListUrls = useActiveListUrls()
   return Boolean(activeListUrls?.includes(url))
 }
+
+export function useGpUnsupportedTokens(): UnsupportedToken | null {
```

Comment:
I thought we would be maintaining a plain list of tokens in a new state, I see you went for further modify the list state. The issue I see is that it makes us diverge more from the original code, so it can bring conflicts and our unsupported list can be mantained separately. 

However, im ok leaving it like that, I guess you analyzed the case and by the shape of this state, you thought is more convenient to use the token list state. Also make sense, I just mention this to make sure you reasoned about it.

Reply:
indeed. i also reasoned that adding all that additional boilerplate for something relatively similar in reponsibility was a bit much. if we are against it down the line im happy to extrapolate it into it's own slice

Reply:
you know better, of course the less we depend on Uniswap code the better. But if you reasoned about this and went for this approach, I won't change it